app/views/layouts/_messages.html.erb

```diff
@@ -3,7 +3,7 @@
   <% if msg.is_a?(String) %>
     <div class="alert alert-dismissible alert-<%= name.to_s == 'notice' ? 'success' : 'danger' %>">
       <button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button>
-      <%= content_tag :div, msg, :id => "flash_#{name}" %>
+      <%= content_tag :div, sanitize(msg), :id => "flash_#{name}" %>
```

Comment:
is this to prevent HTML injection?

Reply:
Yeah it was more force of habit. We're creating a link from the organization name that leads to the invite, and even though GitHub org names should be sanitized to begin with, it doesn't hurt to sanitize this.